.github/workflows/additional-prod-image-tests.yml

```diff
@@ -113,8 +113,6 @@ jobs:
         with:
           fetch-depth: 2
           persist-credentials: false
-      - name: "Cleanup docker"
-        run: ./scripts/ci/cleanup_docker.sh
```

Comment:
Intentional to not call it here anymore?

Reply:
Yes. I realized that the next step

```
      - name: "Prepare breeze & PROD image: ${{ inputs.default-python-version }}"
        uses: ./.github/actions/prepare_breeze_and_image
```

Performs "cleanup_docker" as the first step of the composite action :) . So we were doing cleaning up twice :)